src/odemis/acq/fastem.py

```diff
@@ -207,7 +214,7 @@ def __init__(self, name, coordinates):
         # if None -> acquire, if not None, don't acquire again
 
 
-def acquire(roa, path, scanner, multibeam, descanner, detector, stage, ccd, beamshift, lens, pre_calibrate=False):
+def acquire(roa, path, scanner, multibeam, descanner, detector, stage, ccd, beamshift, lens):
```

Comment:
Since you remove it here you also need to remove it where this function is called, in fastem_acq on line 461.

Reply:
Ah that got added during the rebase again 🙈